stateproofs/stateProofVerifier.go

```diff
@@ -0,0 +1,33 @@
+package stateproofverification
+
+import (
+	"github.com/algorand/go-stateproof-verification/stateproof"
+	"github.com/algorand/go-stateproof-verification/stateproofcrypto"
+
+	"github.com/algorand/go-algorand-sdk/crypto"
+	"github.com/algorand/go-algorand-sdk/encoding/msgpack"
+	"github.com/algorand/go-algorand-sdk/types"
+)
+
+const strengthTarget = uint64(256)
+
+type StateProofVerifier struct {
```

Comment:
I think you should move this code into the `go-stateproof-verification` package.

You can avoid exporting types from that package by aliases the library type in this package:
```
type StateProofVerifier = stateproof.StateProofVerifier
```

Reply:
The rational for including this code in the SDK is allowing users to stick with the SDK (directly at least), rather than adding another library.